tests/functional/instance_manager/__init__.py

```diff
@@ -0,0 +1,120 @@
+# coding=utf-8
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+"""Setup for the package tests."""
+
+import copy
+import json
+import multiprocessing
+import os
+import shlex
+import shutil
+import subprocess
+import sys
+import time
+import uuid
+from subprocess import CalledProcessError
+
+from libtest import get_free_port
+from libtest import project
+from libtest import codechecker
+from libtest import env
+
+# Stopping events for CodeChecker servers.
+EVENT_1 = multiprocessing.Event()
+EVENT_2 = multiprocessing.Event()
+EVENT_3 = multiprocessing.Event()
+
+# Test workspace initialized at setup for authentication tests.
+TEST_WORKSPACE = None
+
+
+def setup_package():
+    """Setup the environment for the tests then start the server."""
+
+    global TEST_WORKSPACE
+    TEST_WORKSPACE = env.get_workspace('instances')
+
+    os.environ['TEST_WORKSPACE'] = TEST_WORKSPACE
+
+    test_config = {}
+
+    # Setup environment varaibled for the test cases.
+    host_port_cfg = env.get_host_port_cfg()
+
+    codechecker_cfg = {
+        'use_postgresql': False,
+        'workspace': TEST_WORKSPACE,
+        'run_names': [],
+        'checkers': []
+    }
+
+    codechecker_cfg.update(host_port_cfg)
+
+    test_config['codechecker_1'] = codechecker_cfg
+
+    # We need a second server
+    codechecker_cfg = {
```

Comment:
We need two unaliased copies of this dict in the `test_config` as they need to configure two separate servers.